packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1446,15 +1457,15 @@ class RawScrollbar extends StatefulWidget {
 /// Provides defaults gestures for dragging the scrollbar thumb and tapping on the
 /// scrollbar track.
 class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProviderStateMixin<T> {
-  Offset? _dragScrollbarAxisOffset;
+  Offset? _startDragScrollbarAxisOffset;
```

Comment:
@TahaTesser Thanks for the sync info, I tested it and it did fix the issue by accident. : )

Reply:
@Piinks Hi, do you have a chance to take a look at this change?  : )

Reply:
Ah yes, apologies! I was out of town and this got buried in my inbox. I checked it out before I left and forgot to click submit - sorry! 😓